src/uct/base/uct_iface.h

```diff
@@ -115,6 +115,42 @@ enum {
         return UCS_ERR_INVALID_PARAM; \
     }
 
+/**
+ * Returns 0 if @a _params field mask does not have the required flags set.
+ */
+#define UCT_EP_PARAMS_CHECK_ADDRS(_params, _flags, _err_message, ...) \
```

Comment:
the name of this macro does not match  the documentation

Reply:
you mean not specific enough?